layouts/page/guideline.html

```diff
@@ -0,0 +1,204 @@
+                <!-- Left Image -->
+                <div class="col-md-2 text-center d-none d-md-block ms-1 me-0 mb-0 px-0 py-0" style="width: 15%">
+                    <img style="max-width: 100%;" src='{{ absURL "guideline/side-img-1.png" }}'/>
+                </div>
+                <!-- Center Content -->
+                <div class="col-md-8">
+                    <h2 class="text-center text-primary fw-lighter">
+                        Know each <span class="fw-bold">department</span>.
+                    </h2>
+                    <p class="text-center">
+                        Find out more about the mission and general tasks of
+                        each department.
+                    </p>
+                </div>
+                <!-- Right Image -->
+                <div class="col-md-2 text-center d-none d-md-block me-1 px-3" style="width: 15%">
+                    <img style="max-width: 100%;" src='{{ absURL "guideline/side-img-2.png" }}' />
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <!--list of departments-->
+    <section class="bg-body-secondary mb-5">
+        <div class="container mt-2 py-5">
+            <!-- Executive Director row -->
+            <div class="row justify-content-center">
+                {{ $depFirstRow := slice "Executive Director" "Secretarial Director" }}
+                {{ range $depFirstRow }}
+                    <div class="col-12 col-md-4 mb-3">
+                        <div style="border-radius: 15px;" class="col bg-secondary py-3 px-3 d-flex align-items-center justify-content-center">
+                            <p class="text-primary mb-0" style="font-weight: 600;">
+                                {{ . }}
+                            </p>
+                        </div>
+                    </div>
+                {{ end }}
+            </div>
+
+            <!-- Department rows -->
+            <div class="row justify-content-center g-3">
+                {{ range $index, $element := $guideline }}
+                    {{ if gt $index 1 }}
+                        <div class="col-md-4">
+                            <div style="border-radius: 15px;" class="col bg-secondary py-3 px-3 d-flex align-items-center justify-content-center">
+                                <p class="text-center mb-0 text-primary" style="font-weight: 600;">
+                                    {{ .department }}
+                                </p>
+                            </div>
+                        </div>
+                    {{ end }}
+                {{ end }}
+            </div>
+        </div>
+    </section>
+
+    <!--info of each departments-->
+    <section class="container mb-5">
+        <div class="py-5 row g-3 justify-content-center">
+            {{ range $guideline }}
+                <div class="col-md-6">
+                    <div class="bg-body-secondary mb-2 border-0 px-3 py-3" style="border-radius: 15px">
```

Comment:
```suggestion
                    <div class="bg-body-secondary mb-2 border-0 p-3 rounded-4">
```

Reply:
bs only provide sizes ranging from 0-3, so if we want to use bs for a larger border radius, i'll have to use the utility api right

Reply:
> bs only provide sizes ranging from 0-3, so if we want to use bs for a larger border radius, i'll have to use the utility api right <img alt="image" width="464" src="https://private-user-images.githubusercontent.com/140220261/351172462-f9d8dc53-1d17-4900-bb4c-4b00ff7e55db.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE3MDQ1MTAsIm5iZiI6MTcyMTcwNDIxMCwicGF0aCI6Ii8xNDAyMjAyNjEvMzUxMTcyNDYyLWY5ZDhkYzUzLTFkMTctNDkwMC1iYjRjLTRiMDBmZjdlNTVkYi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzIzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcyM1QwMzEwMTBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02ZTY5YmIwYjFkYTQxNGExOTM1ZThmNDEwNmE3MTBiZWFjYmM2YTQ4OTBkOTg5MDM5ZGZlZGY5YzBiZGNmNGI4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.9wPlYjCpvsPDkw0BfQ5VhVVFzrZh7CwnEV2gaLaNVe8">

@stingynoodles ure using the wrong version i think

![image](https://github.com/user-attachments/assets/df9d7640-f609-46a9-8d9c-8d52acf0d21f)

Reply:
oh yea my bad thanks